src/main/resources/V4__create_tech_table.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE technology (
```

Comment:
why is this file called tech?
Do we not already have a pr up for the technology table. How can we have this as well?